components/cards/cards.css

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 18.75rem;
+  height: max-content;
+  cursor: pointer;
+}
+
+.card-shadow,
+.card:hover {
+  box-shadow: 13px 13px 21px -5px var(--box-shadow);
+}
+
+.horizontal-card {
+  width: max-content;
+}
+
+.text-overlay-card {
+  width: 17.875rem;
+  height: 8.75rem;
+}
+
+.text-only-card {
+  width: 20rem;
+  padding: 1rem;
+}
+
+.card-image-container {
+  width: 18.75rem;
+  height: 23.438rem;
```

Comment:
you can make it round off , instead of 17.875 and 8.75, you can make then like 18 & 8.5

Reply:
Those were the figma file dimensions given, so tried to use the same